internal/auth/session.go

```diff
@@ -0,0 +1,7 @@
+package auth
+
+// Session is the Realm session
+type Session struct {
```

Comment:
I pulled Session into the auth package and as discussed earlier, if we want to stay with this approach, there are a couple of other items I can pull out as well. We could add just one auth.go file and move structs, vars here. Or we could just pull the whole internal/cloud/auth.go file here